src/cpr_data_access/models.py

```diff
@@ -107,6 +109,18 @@ def _load_and_validate_metadata_csv(
     return metadata_df
 
 
+class KnowledgeBaseIDs(BaseModel):
+    """Store for knowledge base IDs."""
+
+    wikipedia_title: Optional[str]
+    wikidata_id: Optional[Annotated[str, constr(regex=r"^Q\d+$")]]  # type: ignore
+
+    class Config:
+        """Config."""
+
+        frozen = True
```

Comment:
It took me a little while to poke around and figure out what this class was for (I'm assuming that it's here to make `KnowledgeBaseIDs` hashable, as in the [pydantic v1 docs](https://docs.pydantic.dev/1.10/usage/model_config/)?)

Some extra documentation around the hashing logic might be helpful for future readers/users!